src/Monolog/Logger.php

```diff
@@ -285,9 +285,11 @@ public function addRecord($level, $message, array $context = array())
         foreach ($this->processors as $processor) {
             $record = call_user_func($processor, $record);
         }
-        while (isset($this->handlers[$handlerKey]) &&
```

Comment:
IMHO, 

1 - That bubble param is not intention revealing. That should be named 'breakeChain' or something

2- If this bubble thing is what defines this behavior, there should be a method for this property in the [interface](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/HandlerInterface.php#L21)